streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -1699,7 +1705,8 @@ public void shouldReturnOffsetsForRepartitionTopicsForPurging() {
         consumer.assign(asList(partition1, repartition));
         consumer.updateBeginningOffsets(mkMap(mkEntry(repartition, 0L)));
 
-        EasyMock.expect(stateManager.changelogPartitions()).andReturn(Collections.emptySet());
+        EasyMock.expect(stateManager.changelogOffsets()).andReturn(Collections.emptyMap()); // restoration checkpoint
+        EasyMock.expect(stateManager.changelogPartitions()).andReturn(Collections.emptySet()).times(2);
```

Comment:
Since for this test these calls are just stubs (i.e., nothing needs to be verified about these calls), I would replace them with:
```suggestion
EasyMock.expect(stateManager.changelogOffsets()).andStubReturn(Collections.emptyMap()); // restoration checkpoint
        EasyMock.expect(stateManager.changelogPartitions()).andStubReturn(Collections.emptySet());
```

Reply:
Thanks, I think that make sense.